mlflow/langchain/api_request_parallel_processor.py

```diff
@@ -200,7 +201,10 @@ def call_api(
                     try:
                         response = self.lc_model.invoke(
                             self.request_json,
-                            config={"callbacks": callback_handlers},
+                            config={
+                                "callbacks": callback_handlers,
+                                "configurable": self.configurables,
```

Comment:
nit: can we define a `config` variable to avoid repetition?

Reply:
Sure, done